src/handlers/account-deletion-processor-handler.ts

```diff
@@ -0,0 +1,62 @@
+import logger from '../commons/logger';
+import { DynamoDbService as DynamoDatabaseService } from '../services/dynamo-db-service';
+import { LOGS_PREFIX_SENSITIVE_INFO } from '../data-types/constants';
+import { AppConfigService } from '../services/app-config-service';
+import type { Context, SNSMessage, SQSEvent, SQSRecord } from 'aws-lambda';
+import { DeleteStatusUpdateSNSMessage } from '../data-types/interfaces';
+
+const appConfig = AppConfigService.getInstance();
+const ddbService = new DynamoDatabaseService(appConfig.tableName);
+
+export const handler = async (event: SQSEvent, context: Context): Promise<void> => {
+  logger.addContext(context);
+  if (!event.Records[0]) {
+    logger.error('The event does not contain any records.');
```

Comment:
Going to ask what is probably a stupid question so apols, but if the event gets passed in with no records, do we not want to send a metric off to cloudwatch due to an erroneous SQSEvent and throw an error here rather than later on? Or is none of that needed?

Reply:
I think we'll add metrics later on in a separate ticket if necessary :)